test/runnable/foreach5.d

```diff
@@ -1085,6 +1087,45 @@ void test14653()
 }
 
 /***************************************/
+// 15777
+
+template funA15777()
+{
+    import imports.test15777a;
+    alias funA15777 = fun;
+}
+
+template funB15777()
+{
+    import imports.test15777b;
+    alias funB15777 = fun;
+}
+
+template funAB15777()
+{
+    import imports.test15777a;
+    import imports.test15777b;
+    alias funAB15777 = fun;
+}
+
+void foo15777(alias tpl)()
+{
+    alias seq = AliasSeq!(tpl!());
+    // Make alias of 'overload set' in tuple elements
```

Comment:
I don't add such comment, because it's merely a detail of language spec and not related with the bugzilla issue.